graphics/fontforge/Portfile

```diff
@@ -1,17 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-
-name            fontforge
-version         20120731
-revision        3
-set docversion  ${version}
-set ftversion   2.4.10
-categories      graphics fonts
-platforms       darwin
-maintainers     nomaintainer
-license         BSD
-description     Outline and bitmap font editor/converter for many formats
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                fontforge
+github.setup        fontforge fontforge 20170731
+github.tarball_from releases
+use_xz              yes
+distname            ${name}-dist-${version}
+#revision            1
+categories          graphics fonts
+platforms           darwin
+maintainers         nomaintainer
```

Comment:
Would you be willing to become a maintainer?

Reply:
I'm not sure. I have to think about it.